.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
     "extends": "standard",
     "rules": {
-        "indent": ["error", 4]
+        "indent": ["error", 4],
+        "object-shorthand": 0
```

Comment:
Required since we updating eslint required updating the standard config and the standard config started flagging these. I wanted to avoid making a bunch of unrelated changes to fix those for now.